### PR TITLE
fix: drag and drop issue on firefox

### LINKED ISSUE #3094: drag and drop functionalities of elements it's not up to the mark on Firefox.
**Describe the bug**
I wanted to point out on drag and drop functionalities of elements it's not smooth and up to the mark on Firefox.

**To Reproduce**
1. Go to "circuit elements panel" in firefox
2. Click on "any element"
3. Try to drag them 
4. compare with other browsers
5. See error

**Expected behavior**
t is very smooth and easily draggable in chrome and brave but in firefox there is some issue in the response

**Screenshots**

Expected behavior in chrome/ brave-
 
chrome

Glitch in firefox-

firefox

**Are you working on this issue? (Yes/No)**
**No**

### PR DESCRIPTION
**Pull request by Jaagrav**
Fixes #3094 

#### Changes I made in this PR -
This was actually a one line change, `prettier` makes this look like I have made some huge changes. Anyway, the issue as shown in the video on #3094 clearly shows that the image icon was getting dragged on firefox which was causing the drag event listeners to not fire at all. To avoid this, initial steps had already been taken using CSS property `user-drag:none;`, however when I tried it on firefox, I found out that there was no such property supported by firefox as shown in this screenshot.

Firefox doesn't support user-drag property

Long story short, I added a `pointer-events: none;` to the image icon and that did the job perfectly.

### Screencast -
Basically proof that it works...

https://user-images.githubusercontent.com/52719271/171304140-48d09283-b869-442f-8ea2-356a2bf94ed3.mp4

### CODE REVIEW

**Comment by Jaagrav** [comment]
@vedant-jain03 sorry to mention you but maybe the team wasn't notified of this PR? So I'm requesting a review from you.

**Comment by vedant-jain03** [comment]
@Jaagrav Thanks for the PR.
The implementation LGTM but the code change is much large to accept from my perspective.
WDYT @gr455 ?

**Comment by Jaagrav** [comment]
> @Jaagrav Thanks for the PR.
> The implementation LGTM but the code change is much large to accept from my perspective.
> WDYT @gr455 ?

I understand, but it's actually my linter that caused this by adding extra spaces, anyway have fixed it.

File: simulator/src/css/main.stylesheet.css [hunk 1]
```diff
@@ -564,6 +564,8 @@ div.icon img {
   -khtml-user-drag: none;
   -moz-user-drag: none;
   -o-user-drag: none;
+  pointer-events: none;
+
```
Comment (vedant-jain03) [inline simulator/src/css/main.stylesheet.css:568]: Can you please remove this extra line?

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]

File: simulator/src/css/main.stylesheet.css (hunk 1 above)
Comment (Jaagrav) [reply simulator/src/css/main.stylesheet.css]: Done but was it really that necessary? 😅
I mean does it reflect on the website's performance or something that I don't know about?

Comment (vedant-jain03) [reply simulator/src/css/main.stylesheet.css]: Even O(n^3) complexity run and O(n) run the difference in quality, I have learned quality code in CircuitVerse so would always look for quality whether is reflected or not!

**Review by vedant-jain03 — APPROVED** [review]

File: simulator/src/css/main.stylesheet.css (hunk 1 above)
Comment (gr455) [reply simulator/src/css/main.stylesheet.css]: Yep absolutely. We usually have powerful enough hardware to run slightly more unoptimized code, but a developer looking at slightly more messy code can really screw their productivity. Although just an extra newline is not that big of a deal, but code linting overall is pretty important

**Comment by gr455** [comment]
Cool I trust you @vedant-jain03. Merging :tada:
